debezium-connector-postgres/src/main/java/io/debezium/connector/postgresql/PostgresConnectorConfig.java

```diff
@@ -626,6 +626,16 @@ public static SchemaRefreshMode parse(String value) {
                                                         .withImportance(Importance.MEDIUM)
                                                         .withDescription("A name of class to that creates SSL Sockets. Use org.postgresql.ssl.NonValidatingFactory to disable SSL validation in development environments");
 
+    public static final Field SERVER_NAME = Field.create(DATABASE_CONFIG_PREFIX + "server.name")
```

Comment:
Here's a thought: remove the validation from the field in the base class and add two `Field SERVER_NAME` to those connectors actually needing it:

    Field SERVER_NAME = RelationalDatabaseConnectorConfig.SERVER_NAME.withValidation(Field::isRequired, RelationalDatabaseConnectorConfig::validateServerNameIsDifferentFromHistoryTopicName);

This would avoid dupication of description and the other things.

Reply:
I completely agree but I missed that.  I pushed another commit that does exactly that and restores a bit of what we had previously in the `PostgresConnectorConfig`.  Let me know if you see anything else you'd like changed.